curriculum/challenges/english/06-quality-assurance/quality-assurance-projects/metric-imperial-converter.md

```diff
@@ -224,15 +258,15 @@ async (getUserInput) => {
 All 16 unit tests are complete and passing.
 
 ```js
-async (getUserInput) => {
+async getUserInput => {
   try {
     const getTests = await $.get(getUserInput('url') + '/_api/get-tests');
     assert.isArray(getTests);
-    const unitTests = getTests.filter((test) => {
+    const unitTests = getTests.filter(test => {
       return !!test.context.match(/Unit Tests ->/gi);
```

Comment:
```suggestion
      return !!test.context.match(/-> Unit Tests/gi);
```

Reply:
Should we remove the arrow entirely and test only for `Unit Tests`?

Reply:
Either way works for me. The arrow must get added in from the test runner or something.